content/using/running/minio.md

```diff
@@ -0,0 +1,176 @@
+
+The EFF Certbot tool is the easiest way to get a certificate, and [this page](https://certbot.eff.org/instructions) will tell you how to install it depending on your requirements. Select **NGINX** and your operating system for the correct guide.
+
+Once Certbot is installed, you can generate a certificate with the following command:
+
+```
+sudo certbot --server https://acme-v02.api.letsencrypt.org/directory \
+  -d *.s3.example.com \
+  -d s3.example.com \
+  --manual --preferred-challenges dns-01 certonly
+```
+
+After answering a few questions, you will be prompted with a DNS challenge to prove that you own the domains you are creating certificates for. After successfully completing the DNS challenge, your certificate files will be generated and Certbot will print the paths at which they are located. Take note of these for the next step.
+
+Let's Encrypt certificates expire after 90 days, so don't forget to renew them. You should receive an email (to the address you entered when creating the certs) before they are due to expire. To renew, simply repeat the process listed above.
+
+### Install NGINX
+
+Install the NGINX web server and place the following config file at `/etc/nginx/sites-enabled/s3.example.com`. NGINX will act as a proxy to MinIO, allowing us to use our domain and enable TLS.
+
+```
+server {
+    listen 443 ssl;
+    server_name *.s3.example.com;
+
+    ignore_invalid_headers off;
+    client_max_body_size 0;
+    proxy_buffering off;
+
+    ssl_certificate /etc/letsencrypt/live/s3.example.com/fullchain.pem;
+    ssl_certificate_key /etc/letsencrypt/live/s3.example.com/privkey.pem;
+    include /etc/letsencrypt/options-ssl-nginx.conf;
+    ssl_dhparam /etc/letsencrypt/ssl-dhparams.pem;
+
+    location / {
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Proto $scheme;
+        proxy_set_header Host $http_host;
+
+        proxy_pass http://localhost:9000;
+    }
+}
+
+server {
+    listen 80;
+    server_name *.s3.example.com;
+
+    if ($host = *.s3.example.com) {
+        return 301 https://$host$request_uri;
+    }
+
+    return 404;
+}
+```
+
+Update your `server_name` values, ensure that the paths to your certificate files are correct, and change your `proxy_pass` value if you are running MinIO on a different port.
+
+You can test your configuration with `nginx -t`. Any errors will be printed so that you can rectify them.
+
+Restart NGINX to apply the new configuration.
+
+### Create an S3 bucket
+
+Navigate to your root MinIO endpoint (`https://s3.example.com`) in a browser and sign in using the access key and secret key you entered in step 1.
+
+Create a new bucket with the plus button in the bottom right-hand corner of the interface. In the list of buckets on the left hand side of the interface, you should see an option to change the policy to read/write.
```

Comment:
This should make it very clear that you should set the bucket policy to "read only". If it's set to read/write as is currently recommended it means anyone can upload content